carbonmark-api/src/routes/projects/get.ts

```diff
@@ -262,7 +256,7 @@ const handler = (fastify: FastifyInstance) =>
       const singleProject = {
         id: project.id,
         isPoolProject: true,
-        description: cmsData ? cmsData.description.slice(0, 200) : undefined,
+        description: cmsData ? cmsData.description : undefined,
```

Comment:
Apologies the requirements as I described them in #967 were not correct -- I falsely assumed we were delivering `short_description` for all entries here. There seems to be a mistake in the existing code where only user generated listings have a short_description (those in the array on [L123](https://github.com/KlimaDAO/klimadao/pull/1286/files#diff-0c0c1c3310dc142c35bd2519158f3d1a0136f4478324e7e3d2433cee8d93a04bR123)).

But we failed to do the same for PooledProjects here. Looks like we just need to add `short_description` to this `singleProject` object on this line, and then the frontend should already be equipped to [consume it.](https://github.com/KlimaDAO/klimadao/blob/3c912122290faed9a56d36ef3cc367e7fabdc6b2/carbonmark/components/pages/Projects/index.tsx#L74)

The frontend should already

Reply:
I didn't manage to attach the debugger properly and my console.logs also don't get called. Maybe we can take a look together next Monday or Tuesday 😄